REVIEWING.md

```diff
@@ -0,0 +1,67 @@
+# Reviewing Guide
+
+This document covers who may review pull requests for this project, and provides guidance on how to perform code reviews that meet our community standards and code of conduct. All reviewers must read this document and agree to follow the project review guidelines. Reviewers who do not follow these guidelines may have their privileges revoked.
+
+## The Reviewer Role
+
+Only maintainers are REQUIRED to review pull requests.
+Other contributors may opt to review pull requests, but any LGTM from a non-maintainer won't count towards the required number of LGTMs in the Merfify policy.
```

Comment:
:s/Merfify/Mergify/

Also, does LGTM do anything, or should you say clicking Approve?